katsdpdata/prod_handler.py

```diff
@@ -484,6 +503,26 @@ def rdb_file_prefix(self):
         return min(files, key=len)
 
 
+class PrunedProduct(RDBProduct):
+    # Thinking of this as a product might not be technically correct,
+    # but it makes this implementation easier.
+    regex = CAPTURE_BLOCK_REGEX
+
+    def __init__(self, product_path):
+        super().__init__(product_path)
+
+    def metadata_when_created(self):
+        """ When the RDB products are first discovered, we want to set the
+        metadata in SOLR.
+
+        TODO: SET STATES TO FAILED WHEN THIS FAILS
```

Comment:
Do you have a Jira ticket for this work?

Reply:
yes, here is the link  https://skaafrica.atlassian.net/browse/SPR1-2214